### ISSUE TITLE
Endianness Problems

### ISSUE THREAD

**Issue by ashlin4010** [dbrant/QICStreamReader#3]
In your latest commit (2842310) you add configurable endianness, this breaks xenixv3.

The error "inode size seems a bit too large." is thrown before exiting.

Thank You

**Comment by dbrant** [comment]
Hello again!
I'm embarrassed to say that these code scraps aren't really ready for "general" use; this is more like stream-of-consciousness code that I make while reverse-engineering specific data blobs. I was trying to make this program be compatible with all known variants of the Xenix (v3) filesystem, not just the Altos flavor.

Nevertheless, I've fixed the issue you mentioned, so that this program can process your `altos_data` file properly.
Importantly, I added a command-line parameter for specifying the endianness of the blob (which I forgot to add). And also don't forget to specify the offset of the partition. So, your command line should look something like:

`xenixv3 -f altos_data --endianness pdp11 --offset 2818048`

(2818048 being decimal for 0x2B0000, which is where your partition starts. I should really add support for hex offsets, too.)
